var/spack/repos/builtin/packages/recon/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from os import symlink
+
+
+class Recon(MakefilePackage):
+    """RECON: a package for automated de novo identification of repeat families
+       from genomic sequences
```

Comment:
You'll probably want to add a period at the end of this sentence. Last I recall, Spack strips the newline characters and indentation of the docstring in `spack info recon`.

Reply:
Fixed